CSS/aboutme.css

```diff
@@ -0,0 +1,41 @@
+.about-me-bg {
+  flex-direction: column;
+  background-image: url(.././assets/images/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+}
```

Comment:
Was the wifth of the screen 375px? Because on my screen it was fine, just want to make sure

Reply:
Hello @Mhamad-Raad it seems that you are right. You can go try to implement it in order to keep your background image responsive. Meanwhile, I'm gonna make it [OPTIONAL} requirement. Thanks!!

Reply:
Can tou make an image responsive? Wouldnt that break the quality of the background ?

Reply:
You*

Reply:
Yes, you can! Because, if the desktop version will have a minimum width of 600px for example, if you want you can continue with the same background on all screens that have a width which is below 600px. but now it is [OPTIONAL] it will not be considered as the requirement which can block you to move to the next part.

Reply:
Intresting, thank you, will try my best.